### PR TITLE
[12.0] ADD portal_odoo_debranding: removes Odoo branding from portal pages

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

**Review by astirpe — APPROVED** [review]
LGTM

**Review by mrcast — APPROVED** [review]
Functional test ok!

File: portal_odoo_debranding/views/portal_templates.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="frontend_layout_debrand" inherit_id="portal.frontend_layout">
```
Comment (pedrobaeza) [inline portal_odoo_debranding/views/portal_templates.xml:3]: Put priority="9999" for avoiding problems with the replaces

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
See also comments from this PR:

https://github.com/OCA/website/pull/729#discussion_r432573066

File: portal_odoo_debranding/views/portal_templates.xml (hunk 1 above)
Comment (eLBati) [reply portal_odoo_debranding/views/portal_templates.xml:3]: @pedrobaeza I applied what @simahawk suggested at https://github.com/OCA/website/pull/729#discussion_r449424554 as more inheritance compliant.
What do you think?

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**